.gitignore

```diff
@@ -0,0 +1,5 @@
+# Object file
+*.o
+
+# ignore .md files
+test.md
```

Comment:
You've done well to include the `.gitignore` file in your project. However, you need to add the `node_modules` directory in the `.gitignore` file, as it can be added by accident while setting up linters. 

You can think of the node_modules folder as a cache for the external modules that your project depends upon. When you npm install them, they are downloaded from the web and copied into the node_modules folder, and Nodejs is trained to look for them there when you import them (without a specific path). [Read more](https://stackoverflow.com/questions/63294260/what-is-the-purpose-of-node-modules-folder)